### DISCUSSION TITLE
Opening the apk does not properly display the splash screen

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#667] [General Discussion]
Using SDK 33, I have installed the release 4.6 apk.

My apk's splash screen icon never appears correctly when I open it.
Even after adding the SDK 33 patch to my configuration, the problem persists. 

Here is the config file code 
`   <platform name="android">
        <preference name="android-targetSdkVersion" value="33" />
        <hook src="scripts/deleteUnUsableIosIcon.js" type="before_run" />
        <hook src="scripts/deleteUnUsableIosIcon.js" type="before_build" />
        <hook src="scripts/android/fix-android-sdk33-migration.js" type="before_run" />
        <hook src="scripts/android/fix-android-sdk33-migration.js" type="before_build" />
        <hook src="scripts/copyGradleFiles.js" type="before_build" />
        <hook src="scripts/copyBundledData.js" type="before_build" />
        <hook src="scripts/copyGradleFiles.js" type="before_run" />
        <hook src="scripts/copyBundledData.js" type="before_run" />
        <hook src="scripts/android/android-10-migration-fix.js" type="before_build" />
        <hook src="scripts/application/add_application.js" type="before_run" />
        <resource-file src="MainActivity.java" target="app/src/main/java/org/sunbird/app/MainActivity.java" />
        <config-file parent="/manifest/application" target="AndroidManifest.xml">
            <meta-data android:name="io.fabric.ApiKey" android:value="${fabricKey}" />
            <meta-data android:name="io.fabric.ApiSecret" android:value="${fabricSecret}" />
        </config-file>
        <config-file parent="/*" target="AndroidManifest.xml">
            <uses-permission android:name="android.permission.RECORD_AUDIO" />
            <queries>
                <intent>
                    <action android:name="android.media.action.IMAGE_CAPTURE" />
                </intent>
                <intent>
                    <action android:name="android.intent.action.GET_CONTENT" />
                </intent>
            </queries>
        </config-file>
        <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
            <application android:allowBackup="false" android:largeHeap="true" android:networkSecurityConfig="@xml/network_security_config" android:usesCleartextTraffic="true" />
        </edit-config>
        <preference name="android-manifest/@xmlns:tools" value="http://schemas.android.com/tools" />
        <preference name="android-manifest/application/@tools:replace" value="android:allowBackup" />
        <preference name="android-manifest/application/@android:allowBackup" value="false" />
        <preference name="loadUrlTimeoutValue" value="700000" />
        <preference name="CodePushDeploymentKey" value="agojO-OZt4dZlt_pu9r9j2Ipy_jY90dbb065-3633-45a5-9c55-c0405eafaebb" />
        <preference name="AndroidWindowSplashScreenAnimationDuration" value="5000" />
        <preference name="AndroidXEnabled" value="true" />
        <allow-intent href="market:*" />
        <icon density="ldpi" src="resources/android/icon/drawable-ldpi-icon.xml" />
        <icon density="mdpi" src="resources/android/icon/drawable-ldpi-icon.xml" />
        <icon density="hdpi" src="resources/android/icon/drawable-ldpi-icon.xml" />
        <icon density="xhdpi" src="resources/android/icon/drawable-ldpi-icon.xml" />
        <icon density="xxhdpi" src="resources/android/icon/drawable-ldpi-icon.xml" />
        <icon density="xxxhdpi" src="resources/android/icon/drawable-ldpi-icon.xml" />
        <preference name="AutoHideSplashScreen" value="true" />
        <preference name="AndroidWindowSplashScreenBackground" value="#ffffff" />
        <preference name="FadeSplashScreen" value="false" />
        <preference name="SplashScreen" value="resources/android/splash/drawable-ldpi-splash.png" />
        <preference name="SplashScreen" value="resources/android/splash/drawable-hdpi-splash.png" />
        <preference name="AndroidWindowSplashScreenAnimatedIcon" value="resources/android/splash/drawable-ldpi-splash.png" />
        <resource-file src="resources/android/icon/mipmap-hdpi-icon.png" target="app/src/main/res/mipmap-hdpi/n_icon.png" />
        <resource-file src="resources/android/xml/network_security_config.xml" target="app/src/main/res/xml/network_security_config.xml" />
    </platform>`
    
    Here is the image 
    
![Screenshot_2023-12-11-12-19-40-58_6012fa4d4ddec268fc5c7112cbb265e7 (1)](https://github.com/Sunbird-Ed/Community/assets/72926048/a60e0cbe-605b-4b6b-9971-ce81f2cc1190)

cc: @swayangjit @luckynvdu @vpPavithra

**Comment by siddhishinde0723** [comment]
@swayangjit is there any solution for the splash screen icon issue

**Reply by swayangjit** [reply]
@siddhishinde0723  My apologies for the late reply. Are you using the latest splash screen plugin. If yes then the icon size should have following dimensions
Splash screen dimensions
The splash screen icon uses the same specifications as [adaptive icons](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive), as follows:

Branded image: this must be 200×80 dp.
App icon with an icon background: this must be 240×240 dp and fit within a circle 160 dp in diameter.
App icon without an icon background: this must be 288×288 dp and fit within a circle 192 dp in diameter.
For example, if the full size of an image is 300×300 dp, the icon needs to fit within a circle with a diameter of 200 dp. Everything outside the circle turns invisible (masked).

Reference : https://developer.android.com/develop/ui/views/launch/splash-screen

**Reply by siddhishinde0723** [reply]
@swayangjit I tried but still i am facing the same issue for splash screen

**Reply by swayangjit** [reply]
@siddhishinde0723  What is the size of the image svg ?

**Reply by siddhishinde0723** [reply]
@swayangjit The image is 72 X 72 px with .png

**Reply by swayangjit** [reply]
@siddhishinde0723 You have to add  200 * 200 size image.

**Reply by siddhishinde0723** [reply]
@swayangjit I tried with the 200 * 200 as well but still facing the issue.
